src/Symfony/Component/PropertyAccess/PropertyAccessorBuilder.php

```diff
@@ -98,12 +105,36 @@ public function isExceptionOnInvalidIndexEnabled()
     }
 
     /**
+     * Uses a cache system.
+     *
+     * @param Cache|null $cache
+     *
+     * @return PropertyAccessorBuilder The builder object
+     */
+    public function useCache(Cache $cache = null)
```

Comment:
I am not certain that it follows naming notation http://symfony.com/doc/current/contributing/code/conventions.html